contrib/node/src/python/pants/contrib/node/tasks/node_repl.py

```diff
@@ -70,3 +70,9 @@ def launch_repl(self, targets):
 
         repl_session = node_repl.run()
         repl_session.wait()
+    # TODO(qsong): Consider not specifying dependency in BUILD
+    # Repl task doesn't make any sence for Node.js with yarnpkg for the following reasons:
+    # 1. Node.js can simply start from the source root, make an alias for 'cd package_root && node'
+    #   does not give any benefits but increase the maintance work
+    # 2. There's no simple entry point (binary) for Node.js packages. A package may start with node,
```

Comment:
> it should be a separate PR in my POV.

+1. But if you do want to allow an entrypoint to be specified, that's easy to do as an option. The python repl, for example, allows [configuring which repl entrypoint](https://github.com/pantsbuild/pants/blob/master/src/python/pants/backend/python/tasks/python_repl.py#L19) to use.